app/scripts/react-directives/components/modals/formModal.jsx

```
@@ -12,18 +12,27 @@ import { MakeFormFields } from '../../forms/forms';
         <ModalTitle id={id} text={title} />
       </ModalHeader>
-      <ModalBody>{MakeFormFields(Object.entries(formStore?.params || {}))}</ModalBody>
+      <ModalBody>
+        <form id={formId} onSubmit={onSubmit}>
+          {MakeFormFields(Object.entries(formStore?.params || {}))}
+        </form>
+      </ModalBody>
       <ModalFooter>
         <Button
           label={okButtonLabel}
           type={'success'}
```

ESLint

[error] Curly braces are unnecessary here.

(react/jsx-curly-brace-presence)